modin/pandas/base.py

```diff
@@ -3856,6 +3856,30 @@ def values(self):  # noqa: RT01, D200
         """
         return self.to_numpy()
 
+    def _repartition(self, axis: Optional[int] = None):
```

Comment:
We should probably add _repartition in to _DEFAULT_BEHAVIOUR in this file.

Reply:
added